drivers/hmis/app/graphql/mutations/unassign_staff.rb

```diff
@@ -0,0 +1,26 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
+  class UnassignStaff < CleanBaseMutation
+    argument :id, ID, required: true
+
+    field :staff_assignment, Types::HmisSchema::StaffAssignment, null: true
+
+    def resolve(id:)
+      record = Hmis::StaffAssignment.preload(household: :project).find_by(id: id)
+      raise 'Not found' unless record
+
+      access_denied! unless current_user.permissions_for?(record.household.project, :can_edit_enrollments)
```

Comment:
I think it might be better to standardize on the `current_permission?` helper

```suggestion
      access_denied! unless current_permission?(permission: :can_edit_enrollments, entity: record.household.project)
```